gethrpc/subscription.go

```diff
@@ -170,7 +170,7 @@ func (n *Notifier) activate() error {
 }
 
 func (n *Notifier) send(sub *Subscription, data json.RawMessage) error {
-	params, _ := json.Marshal(&subscriptionResult{ID: string(sub.ID), Result: data})
+	params, _ := json.Marshal(&subscriptionResult{ID: 1, Result: data}) // TODO
```

Comment:
what's the TODO here? 
Its better to have a descriptive comment on TODOs :)

Reply:
Absolutely, added a comment